starport/cmd/network_chain_publish.go

```diff
@@ -135,14 +120,8 @@ func networkChainPublishHandler(cmd *cobra.Command, args []string) error {
 			}
 			s.Start()
 		}
-
-		if err := blockchain.Init(cmd.Context()); err != nil {
```

Comment:
We were intentionally calling the Init func here manually so that way we could show the `Publishing...` spinner before we run the `Publish` command to show the current action to the user. Can we take this change back and also take back the removed line for `s.SetText("Publishing...")`?

Reply:
I refactored a bit. I needed to put the genesis option in `Blockchain()` constructor so that we don't download it twice in case of check needed